exercises/concept/paolas-prestigious-pizza/.docs/introduction.md.tpl

```diff
@@ -0,0 +1,3 @@
+# Introduction
+
+%{concept: parsing}
```

Comment:
This is a pretty cool new feature, running `bin/configlet generate` will find this file and create a whole new `introduction.md` that is a copy of the one in the `concepts/parsing` folder.
However, I must remember to run it once the review is done.